### DISCUSSION TITLE
Lf won't update after a file added to a directory

### DISCUSSION THREAD

**Discussion by LinArcX** [gokcehan/lf#909] [Q&A] [answered]
Hi.

I'm using Lf on void Linux with wayfire window manager.

I'm also using grim+slurp to take screenshots.

After I take a screenshot, I look at `~/Pictures`, And nothing there. I should always run `:reload` to see new files.

Is there any workaround for this?

**Comment by lahwaacz** [comment, marked as answer]
Search for the `period` option in the documentation: https://pkg.go.dev/github.com/gokcehan/lf#section-documentation

**Reply by lahwaacz** [reply]
Alternatively, you may want to add `lf -remote "send reload"` to your screenshot taking script.
